js/main.ts

```diff
@@ -0,0 +1,49 @@
+import {getRandomArrayElement, getRandomInt} from './utils/randomizers.ts';
+import {User, PhotoComment, Photo} from './contracts/common.ts';
+import {messages, names, descriptions} from './data/template-values.json';
+
+export const getGeneratedMocks = (amount: number):Photo[] => {
+	enum Default {
+		MinId = 1,
+		MinPhotoAddress = 1,
+		MaxPhotoAddress = 25,
+		MinLikes = 15,
+		MaxLikes = 200,
+		MinComments = 0,
+		MaxComments = 30,
+		MinAvatarId = 1,
+		MaxAvatarId = 6
+	}
+	const getId = () => {
+		let id = Default.MinId;
+		return () => id++;
+	};
+	const getCommentId = getId();
+	const getPhotoId = getId();
+	const _getGeneratedUser = ():User => ({
+		avatar: `img/avatar-${getRandomInt(Default.MinAvatarId,Default.MaxAvatarId)}.svg`,
+		name: getRandomArrayElement(names)
+	});
+
+	const _getGeneratedComment = ():PhotoComment => {
+		const user:User = _getGeneratedUser();
+		return {
+			avatar: user.avatar,
+			id: getCommentId(),
+			message: Array.from({length: getRandomInt(1, 2)}, () => getRandomArrayElement(messages)).join(' '),
+			name: user.name
+		};
```

Comment:
Если не боишься неявного чтения

```suggestion
		return {
			...user,
			id: getCommentId(),
			message: Array.from({length: getRandomInt(1, 2)}, () => getRandomArrayElement(messages)).join(' '),
		};
```